### ISSUE TITLE
What’s needed to support mobile devices?

### ISSUE THREAD

**Issue by claremacrae** [Robin-Haupt-1/Obsidian-Map-of-Content#31]
Is it known what would need to be done to make this plugin support mobile devices such as iPads?

I would love to be able to help.

**Comment by Robin-Haupt-1** [comment]
That's something i would like too. I don't think there's anything really stopping it from working on mobile now, except i don't know how to get the view to open on mobile, so i set it on desktop only for now. I've only tried it on my Android phone, it might even work on tablets. I'll try that when i get a chance

**Comment by claremacrae** [comment]
Thank you Robin.

**Comment by Robin-Haupt-1** [comment]
I've released a new version that can be installed on mobile you can try it out too. I still can't see the view on my tablet though.

**Comment by Robin-Haupt-1** [comment]
Either im missing something or normal views can't be seen on mobile. Ill try creating a modal window that can be opened from the command palette to display the MOC

**Comment by claremacrae** [comment]
Thanks. I don’t know the vocabulary of the Obsidian layout - but on my ipad it shows up on the options on the right hand side of the screen - and I can view it and pin it!

**Comment by claremacrae** [comment]
How to view it:

![DC099978-2043-4F5B-960C-7F3612C5330F](https://user-images.githubusercontent.com/4840096/136169536-21f6cd91-a196-4369-87e8-b6b8ea343388.jpeg)

What it looks like initially:

![17360D6A-F675-4492-A4EB-E4A3C47A658F](https://user-images.githubusercontent.com/4840096/136169567-ce3c287b-7ca3-4d0f-9b36-33a3b421228c.jpeg)

What it looks like when pinned:

![BFC64609-643E-4771-99E0-BBE59B56EED8](https://user-images.githubusercontent.com/4840096/136169594-4ff2daf2-efca-46ef-85e5-945322401d43.jpeg)

**Comment by Robin-Haupt-1** [comment]
I'm glad it works on some devices. i have created a thread in the Obsidian forum to get some more information: https://forum.obsidian.md/t/how-to-display-a-view-on-mobile/25256/2

On my Android tablet it doesn't show anything where it says "Map of Content" on yours. I tried installing the dangling links plugin you use and that doesn't show either.

**Comment by kometenstaub** [comment]
Does a restart of the android app help? (closing, swiping it away, reopening) 

Which android version are you using?

**Comment by Robin-Haupt-1** [comment]
I'm using version 1.0.4 of the app on my phone with runs Android 10 and my Fire tablet which runs Fire OS 7 which is based on Android 9. Restarting the app does not help.

Do you also see it where claremacrae sees it?

**Comment by kometenstaub** [comment]
Yes, it is also there for me. Did you activate the plugin and set the central node? How do you sync it to your android devices?
